### PR TITLE
wavebox: 4.11.3->10.114.26-2

### PR DESCRIPTION
**Pull request by eddsteel**
###### Description of changes

Updates wavebox to the latest stable version, and adds me as a maintainer.

Since v4 there have been significant changes, for example moving the core engine to chromium not electron. Also the licensing has changed and the release artifact location has moved. 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by eddsteel** [comment]
would you mind taking a look @rawkode ?

**Comment by Kukkerem** [comment]
HI @eddsteel, I somehow missed this PR and created a new one: https://github.com/NixOS/nixpkgs/pull/209140
Please check/merge my modifications (maybe the version upgrade, icon creation or wavebox-launcher usage).
I had no intention of being a maintainer, so I will delete it soon if you are willing to be one.
Thank you for your work! :)

**Comment by eddsteel** [comment]
Thanks @Kukkerem ! I've merged your improvements here. I'm willing to be maintainer if there isn't one, unfortunately I still can't actually merge this, yet.

I think we still need @rawkode 's help to get this merged.

**Review by rawkode — APPROVED** [review]

**Comment by rawkode** [comment]
Approved. Apologies, this slipped my notifications

**Comment by eddsteel** [comment]
Thanks a lot @rawkode ! Now, who can actually merge it?

**Comment by Kukkerem** [comment]
@eddsteel, could you resolve the conflicts? @rawkode, could you merge it when the conflicts are resolved?

**Comment by eddsteel** [comment]
Bumped the version again and resolved conflicts.

File: pkgs/applications/networking/instant-messengers/wavebox/default.nix
Comment (eddsteel) [inline pkgs/applications/networking/instant-messengers/wavebox/default.nix:61]:
```suggestion
  runtimeDependencies = [ (lib.getLib udev) libnotify gtk4 ];
```

**Comment by eddsteel** [comment]
@rawkode please merge if you think this looks OK

File: pkgs/applications/networking/instant-messengers/wavebox/default.nix
Comment (eddsteel) [reply pkgs/applications/networking/instant-messengers/wavebox/default.nix:40]: Thanks, I'll update to the latest version.

**Review by cole-h — APPROVED** [review]
LGTM, thanks! Ran just fine.

**Comment by eddsteel** [comment]
@cole-h thanks so much! I don't currently have write access and need someone else to merge this.

**Comment by cole-h** [comment]
I'll happily merge this once ofborg is green :)

(The queue is somewhat large and slow at the moment, sorry about that.)

**Pull request merged**
